File: farandula-java/src/main/java/com/nearsoft/farandula/models/Itinerary.java
package com.nearsoft.farandula.models;

import java.util.ArrayList;
import java.util.List;

/**
 * Created by pruiz on 5/11/17.
 */
public class Itinerary {

    private List<AirLeg> airlegs = new ArrayList<>();
    private Fares price;

    public List<AirLeg> getAirlegs() {
        return airlegs;
    }

    public void setAirlegs( List<AirLeg> airleg) {
        this.airlegs = airlegs;
    }

    public Fares getPrice() {
        return price;
    }

    public void setPrice(Fares price) {
        this.price = price;
    }

    @Override
    public String toString() {
        return "Itinerary{" +
                "airlegs=" + airlegs +
                ", price=" + price +
                '}';
    }
}

File: farandula-ruby/lib/farandula/models/itinerary.rb
module Farandula
  class Itinerary

    attr_accessor :id
    attr_accessor :air_legs
    attr_accessor :fares

    def initialize(
      id = nil,
      air_legs  = [],
      fares     = nil
    )

      @id       = id
      @air_legs = air_legs
      @fares    = fares

    end

    def to_s
      result = ""
      result << "id #{id}, " \
                "air_legs #{air_legs.to_s}, " \
                "fares #{fares}."
    end

  end 
end